site/src/pages/ManagementSettingsPage/OrganizationMembersPage.tsx

```diff
@@ -22,6 +23,8 @@ const OrganizationMembersPage: FC = () => {
 	};
 	const { user: me } = useAuthenticated();
 
+	const groupsByUserIdQuery = useQuery(groupsByUserId());
```

Comment:
note how this query doesn't take the `organizationName` as a parameter: this is not scoped to an organization, this shows _every_ group a user is in.